docs/secure-connections/sauce-connect/system-requirements.md

```diff
@@ -145,373 +145,134 @@ As an important step prior to downloading Sauce Connect Proxy, you or your syste
 
 ## Configuring Your System to Use Sauce Connect
 
-Select a cloud provider from the tables below to view the recommended system requirements:
+Select a cloud provider from the tables below to view the minimum and recommended system requirements:
+
+:::note These are general recommendations; actual system requirements will depend on the number of parallel tests, and amount of data transferred with each test.
```

Comment:
```suggestion
:::note
These are general recommendations; actual system requirements will depend on the number of parallel tests, and amount of data transferred with each test.
:::
```

Reply:
^ this suggested change will display like this:

<img width="798" alt="Screenshot 2023-01-13 at 4 55 31 PM" src="https://user-images.githubusercontent.com/15352964/212426538-12ee8b99-1acd-4b22-a580-4a64ac3b4a9e.png">